tests/cephadm/test_cephadm_log_spam.py

```diff
@@ -0,0 +1,29 @@
+from cli.exceptions import OperationFailedError
+
+
+def run(ceph_cluster, **kw):
+    """Verify spam logs not present in cephadm.log
+    Args:
+        **kw: Key/value pairs of configuration information
+              to be used in the test.
+    """
+    node = ceph_cluster.get_nodes(role="_admin")[0]
+    CEPHADM_LOG_PATH = "/var/log/ceph/cephadm.log"
+
+    client = node.rssh if kw.get("sudo", False) else node.ssh
+    with client().open_sftp().open(CEPHADM_LOG_PATH, "r") as file:
```

Comment:
You can use download_file method instead.
Refer: https://github.com/red-hat-storage/cephci/blob/master/tests/cephfs/nfs/test_nfs_mount.py#L183

See if that can be reused.

Reply:
I would just need to read the contents of the file hence using open() and read() here.
Downloading the file would be another extra step as after downloading I would need to again read it.